frontend/packages/pipelines-plugin/src/components/catalog/providers/useTasksProvider.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { TFunction } from 'i18next';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { CatalogItem, ExtensionHook } from '@console/dynamic-plugin-sdk';
+import { getImageForIconClass } from '@console/internal/components/catalog/catalog-item-icon';
+import { useK8sWatchResources } from '@console/internal/components/utils/k8s-watch-hook';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { TaskModel, ClusterTaskModel } from '../../../models';
+import { TaskKind } from '../../../types';
+import { TektonTaskAnnotation, TektonTaskLabel } from '../../pipelines/const';
+
+const normalizeTektonTasks = (TektonTasks: TaskKind[], t: TFunction): CatalogItem<TaskKind>[] => {
```

Comment:
```suggestion
const normalizeTektonTasks = (tektonTasks: TaskKind[]): CatalogItem<TaskKind>[] => {
```

It's a variable, not a class/component name, camelCase please.

Also don't pass `t` into methods.